lcls-twincat-motion/Library/DUTs/DUT_MotionStage.TcDUT

```diff
@@ -1,59 +1,224 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.12">
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
   <DUT Name="DUT_MotionStage" Id="{10b5775d-5646-4d55-ad77-40abc3888de3}">
     <Declaration><![CDATA[TYPE DUT_MotionStage :
 STRUCT
-	// Hardware
+	(* Hardware *)
+	
+	// PLC Axis Reference
 	Axis: AXIS_REF;
+	// NC Forward Limit Switch: TRUE if ok to move
+	{attribute 'pytmc': '
+		pv: IO:ENA:FWD
```

Comment:
Just an observation: the final suffix is uppercase in these, but camelcase in the settings section.